### PR TITLE
tests: Define who's responsible for what on the team

### PR DESCRIPTION
**Pull request by atodorov**
because we're expecting to onboard other people as well and we need
to start keeping track of this.

@jrusz, @henrywang, @yuxisun1217 please help me define/refine your responsibilities better!

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -309,6 +309,18 @@ identified and with the mutual understanding that PRs related to
...
 
+## QE associates/responsibilities matrix
+
+- [@atodorov](https://github.com/atodorov) - QE lead;
+  osbuild, osbuild-composer, weldr-client, composer-cli, image-builder, image-builder-frontend,
+  bug testing & verification, anything else
+- [@jrusz](https://github.com/jrusz) - testing across the board; backup for atodorov;
+  osbuild, osbuild-composer, weldr-client, composer-cli, image-builder, image-builder-frontend,
+  bug testing & verification, anything else
+- [@henrywang](https://github.com/henrywang) - everything related to testing RHEL for Edge images
```
Comment (henrywang) [inline test/README.md:320]: @atodorov Could you please add @yih-redhat in this list? He's another RHEL for Edge project QE. Thanks.

[@yih-redhat](https://github.com/yih-redhat) - everything related to testing RHEL for Edge images; backup for @henrywang

Comment (atodorov) [reply test/README.md:320]: Added. It probably makes sense to also add them to our QE meeting. See your updated invite for a new timeslot which serves better your timezone.

**Comment by ondrejbudai** [comment]
I wonder if this is the right place because this matrix spans multiple projects, not just osbuild-composer. Maybe let's have this information on osbuild.org?

**Comment by atodorov** [comment]
> I wonder if this is the right place because this matrix spans multiple projects, not just osbuild-composer. Maybe let's have this information on osbuild.org?

While that makes sense there is also lots of other related information in this README which is valid for the other projects as well.  I am open to suggestions for a better home for this but for now let's keep it in test/README together with the rest if it.

**Review by ondrejbudai — APPROVED** [review]

**Pull request merged**